internal/claude/parser.go

```
@@ -410,6 +466,11 @@ func findParentRepository(directory string) string {
 	return ""
 }
 
+// findParentRepository walks up the directory tree to find a parent git repository
+func findParentRepository(directory string) string {
```

golangci-lint (1.64.8)

func `findParentRepository` is unused

(unused)